### PR TITLE
Add the google reports drive documentation

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
- Add Google Drive Reports Documentation

### CODE REVIEW

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Google Reports is a data reporting and analysis platform offered by Google, designed to provide insights and metrics about user activities and interactions within various Google services. It allows organizations to track and visualize user engagement, application usage, and other relevant data points, enabling informed decision-making and optimization of digital experiences. And Google Reports Drive is just for the Google Drive service.
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_bb11239e-3dc6-11ee-be56-0242ac120002_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/bb11239e-3dc6-11ee-be56-0242ac120002.md!}
+
+## Google Cloud configuration procedure
+
+#### Prerequisites
```
Comment (Adamowoc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:15]:
```suggestion
### Prerequisites
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+- Google licence Enterprise standard or higher
+- Access to Sekoia.io Intakes and Playbook pages with write permissions
+- Administrator access to the Google Cloud console
+
+#### Create a dedicated service account
```
Comment (Adamowoc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:21]:
```suggestion
### Create a dedicated service account
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+To create a service account you have to :
+ - Create a project
+ - Turn on the APIs for the service account
+ - Set up the OAuth consent screen
+ - Create the service account
+
+For more details in each steps please read this [Documentation](https://support.google.com/a/answer/7378726?hl=en)
+
+#### Create and download JSON keys (service account credentials)
```
Comment (Adamowoc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:31]:
```suggestion
### Create and download JSON keys (service account credentials)
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://accounts.google.com/o/oauth2/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/SERVICE_ACCOUNT_EMAIL"
+}
+```
+
+## Sekoia.io configuration procedure
+
+#### Create your intake
```
Comment (Adamowoc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:69]:
```suggestion
### Create your intake
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+1. Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the Google drive reports.
+2. Copy the associated Intake key
+
+#### Pull the logs to collect them on Sekoia.io
```
Comment (Adamowoc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:74]:
```suggestion
### Pull the logs to collect them on Sekoia.io
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+
+-  Create a **Trigger configuration** using:
+
+	* `Intake key` created on the previous
+
+- Click on the **Save** button
+- **Activate the playbook** with the toggle button on the top right corner of the page
+
+#### Enjoy your events on the [Events page](https://app.sekoia.io/operations/events)
```
Comment (Adamowoc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:92]:
```suggestion
### Enjoy your events on the [Events page](https://app.sekoia.io/operations/events)
```

**Review by Adamowoc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+- Access to Sekoia.io Intakes and Playbook pages with write permissions
+- Administrator access to the Google Cloud console
+
+### Create a dedicated service account
+
+To create a service account you have to :
+ - Create a project
+ - Turn on the APIs for the service account
+ - Set up the OAuth consent screen
+ - Create the service account
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:27]: Add another line between the previous sentence and the list for the layout
```suggestion

 - Create a project
 - Turn on the APIs for the service account
 - Set up the OAuth consent screen
 - Create the service account
```

**Review by squioc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/SERVICE_ACCOUNT_EMAIL"
+}
+```
+
+## Sekoia.io configuration procedure
+
+### Create your intake
+
+1. Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the Google drive reports.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/cloud_and_saas/google/google_drive_reports.md:75]: I forgot that I changed the name of the format
```suggestion
1. Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the `Google Report`.
```

**Review by squioc — CHANGES_REQUESTED** [review]

**Review by squioc — APPROVED** [review]

**Pull request merged**